pkg/integrations/manager_test.go

```diff
@@ -24,12 +24,13 @@ import (
 // remarshaled back out to text.
 func TestConfig_Remarshal(t *testing.T) {
 	RegisterIntegration(&testIntegrationA{})
-
 	cfgText := `
 scrape_integrations: true
 replace_instance_label: true
 integration_restart_backoff: 5s
 use_hostname_label: true
+http_tls_config:
+  insecure_skip_verify: false
```

Comment:
(Just as a note, you had to do this because none of our yaml fields are specified as omit empty, which is something we should fix eventually, but not right now)